src/Controllers/GagePageController.ts

```diff
@@ -200,7 +204,11 @@ module StreamStats.Controllers {
         public formattedDailyPlusAvg = [];
         public formattedEstPeakDates = [];
         public formattedDailyFlow = [];
+        public dailyRange = [];
```

Comment:
I don't think this variable is being used & can be removed.

Reply:
I think I had started work on creating sliders and I would need this to help define time spans. But I can delete it for this push!